.gitmodules

```diff
@@ -0,0 +1,4 @@
+[submodule "crypto"]
```

Comment:
The intention is to eventually get even `mbedtls_*` crypto from here, not just PSA crypto. I've renamed the `ENABLE_PSA` build system option to `USE_CRYPTO_SUBMODULE` to better reflect the intent, which is to use libmbedcrypto as built by the crypto submodule instead of the `mbedtls` repo's copy.

Reply:
My problem is we're moving from one Crypto interface to another, and I think it should be apparent. It's not just that we're *moving* the crypto files, but we're dropping one crypto API for another. The name blurs this.

Ultimately, it's your call.

Reply:
The API drop will happen after the submodule switch, not during or before. Switching to the submodule must not break API compatibility yet.